mlflow/tracing/fluent.py

```diff
@@ -294,6 +303,12 @@ def _end_stream_span(
                 outputs = output_reducer(outputs)
             except Exception as e:
                 _logger.debug(f"Failed to reduce outputs from stream: {e}")
+
+        try:
+            set_chat_attributes_special_case(span, inputs=inputs, outputs=outputs)
+        except Exception:
+            pass
```

Comment:
Do we need try catch here? set_chat_attributes_special_case already has try catch (similar for other places)